<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="1">
     <widget class="QComboBox" name="combo_device"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="combo_profile"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Device</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Profile</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="HardwareViewWidget" name="deviceView"/>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QGroupBox" name="actionGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>250</height>
       </size>
      </property>
      <property name="title">
       <string>Action Settings</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="actionSettingWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>679</width>
            <height>217</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2" colspan="2">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>343</width>
         <height>69</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QStackedWidget" name="device_param_widget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionDocks"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="ActionList">
   <property name="windowTitle">
    <string>Actions</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="actionDock">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="action_list_widget">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Category</string>
        </property>
        <item>
         <property name="text">
          <string>Subcategory</string>
         </property>
         <item>
          <property name="text">
           <string>Action 0</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Action 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Action 2</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>Category 2</string>
        </property>
        <item>
         <property name="text">
          <string>Action 3</string>
         </property>
        </item>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionDocks">
   <property name="text">
    <string>Docks</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HardwareViewWidget</class>
   <extends>QGraphicsView</extends>
   <header>virtualstudio.configurator.ui.widgets.hardwareview</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
